docs/en/table_design/indexes/Bloomfilter_index.md

```diff
@@ -20,7 +20,10 @@ For example, you create a bloom filter index on a `column1` of a given table `ta
 ## Usage notes
 
 - You can create bloom filter indexes for all columns of a Duplicate Key or Primary Key table. For an Aggregate table or Unique Key table, you can only create bloom filter indexes for key columns.
-- TINYINT, FLOAT, DOUBLE, and DECIMAL columns do not support creating bloom filter indexes.
+- The Bloom filter indexes can be created for the columns of the following types:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/77648488-4958-423f-84d4-1ab43ce350f6)
remove "The"